apiary.apib

```diff
@@ -3289,6 +3289,81 @@ Download a credit note in a specific format.
             + location: `https://cdn.teamleader.eu/file` (string) - A temporary url where the requested file can be downloaded
             + expires: `2018-02-05T16:44:33+00:00` (string) - Expiration time of the temporary download link
 
+## Subscriptions [/subscriptions]
+
+### subscriptions.list [GET /subscriptions.list]
+
++ Request (application/json)
+    + Attributes (object)
+        + filter (object, optional)
+            + ids: `46156648-87c6-478d-8aa7-1dc3a00dacab`, `46156648-87c6-478d-8aa7-1dc3a00daca4` (array[string], optional)
+            + invoice_id: `60f03745-4289-4ae7-8c74-d797860bc887` (string, optional)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum, required)
+                    + Members
+                        + title
+                        + created_at
+                + order (enum, optional)
+                    + Members
+                        + asc
+                        + desc
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `e2314517-3cab-4aa9-8471-450e73449041` (string)
+                + department (object)
+                    + type: `department` (string)
+                    + id: `5e90eb0a-b502-4344-aa0f-3b8525af6186` (string)
+                + title: `Subscription for cookies` (string)
+                + active: true (boolean)
+                + next_renewal_date: `2020-06-21` (string)
+                + invoicee (object)
```

Comment:
Q: I don't think `invoicee` is the correct term here since there is no invoice, what about `subscriber`?

Reply:
`invoice recipient` is literally the description when you look up the term `invoicee`.
To me, this does not feel like an improvement.

`Subscriber`, on the other side, is more into context. Let's check with @Miljar on this.

Reply:
It's the invoicee on the invoices that are generated, seems fine to me.

Reply:
> `invoice recipient` is literally the description when you look up the term `invoicee`.

Ya, I realised that as well yesterday after closing my laptop 😂.

Reply:
@tiagobastos also noticed it is in line with the object (containing a **customer and FAO**) that is returned in `invoices.list`